test/skip-options.test.js

```diff
@@ -58,41 +58,68 @@ t.test('should not apply the delay header', async t => {
     })
     t.teardown(() => fastify.close())
 
+    const test = {
```

Comment:
why an object and not a simple counter?

Reply:
because closure will remember the value when it is created :). However it's not the case when you are using references